README.md

```diff
@@ -855,3 +855,93 @@ Out[6]:
  'total_bytes': 16,
  'total_count': 1}
 ```
+
+## Taking ownership of C++ objects from Python.
+
+When interacting with a C++ library that uses RMM from Python, one
+must be careful when taking ownership of `rmm::device_buffer` objects
+on the Python side. The `rmm::device_buffer` contains a _raw_ pointer
+to the memory resource used for its allocation, and the allocating
```

Comment:
I think this is no longer true, it's holding a `device_async_resource_ref` now, right?

Reply:
Good point, done